c_glib/arrow-glib/basic-data-type.cpp

```diff
@@ -1493,16 +1495,20 @@ garrow_decimal_data_type_class_init(GArrowDecimalDataTypeClass *klass)
  * Returns: (nullable):
  *   The newly created decimal data type on success, %NULL on error.
  *
- *   #GArrowDecimal256DataType is used if @precision is larger than
- *   garrow_decimal128_data_type_max_precision(),
- *   #GArrowDecimal128DataType is used otherwise.
+ *   #GArrowDecimal32DataType is used if @precision up to 9
+ *   #GArrowDecimal64DataType is used if @precision up to 19
+ *   #GArrowDecimal128DataType is used if @precision up to 38
+ *   Otherwise, #GArrowDecimal256DataType is used
```

Comment:
How about using list?

```suggestion
 *   * #GArrowDecimal32DataType is used if @precision up to 9
 *   * #GArrowDecimal64DataType is used if @precision up to 19
 *   * #GArrowDecimal128DataType is used if @precision up to 38
 *   * #GArrowDecimal256DataType is used otherwise
```

Reply:
Thanks Fixed.